zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -310,9 +327,12 @@ impl NetworkUpgrade {
         network
             .activation_list()
             .iter()
-            .filter(|(_, nu)| nu == &self)
+            .find(|(_, nu)| nu == &self)
             .map(|(height, _)| *height)
-            .next()
+            .or_else(|| {
+                self.next_upgrade()
+                    .and_then(|next_nu| next_nu.activation_height(network))
+            })
```

Comment:
I think this change merits a docs update.

Reply:
Updated the docs in https://github.com/ZcashFoundation/zebra/pull/8383/commits/fb2d13a7062d5d65db49cf89a472425c9efd59bd.